paddle/fluid/inference/analysis/helper.h

```diff
@@ -153,25 +153,6 @@ T &GetFromScope(const framework::Scope &scope, const std::string &name) {
   return *var->GetMutable<T>();
 }
 
-static framework::proto::ProgramDesc LoadProgramDesc(
```

Comment:
Delete since this function is unused.  The duplicate implementations  in  ./paddle/fluid/inference/api/analysis_predictor.h .  (Only for review )